examples/text_summarization/bertsum_cnndm.ipynb

```diff
@@ -0,0 +1,3921 @@
+{
```

Comment:
<p>"BertSum refers to [Fine-tune BERT for Extractive Summarization" missing right square bracket.</p><p>missing "n" in "give summary"</p><p>typo in "simplfied to classifying..."</p><p>Remove "whether" in  "predicts whether the probability"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp/pull/447/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>